amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/SynchronousBlockingRetrievalFactory.java

```diff
@@ -79,42 +63,27 @@ public SynchronousBlockingRetrievalFactory(String streamName,
                 defaultDataFetcherProvider(kinesisClient) : dataFetcherProvider;
     }
 
-    @Deprecated
-    public SynchronousBlockingRetrievalFactory(String streamName,
-                                               KinesisAsyncClient kinesisClient,
-                                               RecordsFetcherFactory recordsFetcherFactory,
-                                               int maxRecords) {
-        this(streamName, kinesisClient, recordsFetcherFactory, maxRecords, PollingConfig.DEFAULT_REQUEST_TIMEOUT);
-    }
-
     private static Function<DataFetcherProviderConfig, DataFetcher> defaultDataFetcherProvider(
             KinesisAsyncClient kinesisClient) {
         return dataFetcherProviderConfig -> new KinesisDataFetcher(kinesisClient, dataFetcherProviderConfig);
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
-    public GetRecordsRetrievalStrategy createGetRecordsRetrievalStrategy(@NonNull final ShardInfo shardInfo,
```

Comment:
:( dont know why this was part of the public interface, thanks for marking it deprecated, its never called from outside and its only called from within the class so it dint need to be part of the public interface, could just have been a private method.